di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -77,12 +77,13 @@ public class IpvHandler {
                 String errorType = request.queryMap().get("error").value();
                 String userIdTextValue = request.queryMap().get("userIdText").value();
                 String signInJourneyIdText = request.queryMap().get("signInJourneyIdText").value();
+                String[] vtr = request.queryMap().get("vtrText").values();
```

Comment:
Does this work if you try and enter multiple values into the `vtr` - what's the format for that?

Reply:
For example
 Cl.Cm.P2,Cl.Cm.PCL200,Cl.Cm.P1

Reply:
Nice :)